include/ck/tensor_operation/gpu/grid/gridwise_gemm_multiple_d_xdl_cshuffle.hpp

```diff
@@ -92,6 +92,13 @@ struct GridwiseGemmMultipleD_xdl_cshuffle
     using GridwiseGemmPipe = remove_cvref_t<decltype(
         GridwiseGemmPipeline_Selector<PipelineVer, NumGemmKPrefetchStage, LoopSched>())>;
 
+    // denorm test fix, required to work around fp16 mfma issue
+    // we convert fp16->fp32->bf16 and execute bf16 mfma instruction
+    // when mfma if fixed, remove this section and update
+    // ABDataTypeAdjusted -> ABDataType throughout this file
+    using ABDataTypeAdjusted =
```

Comment:
We can have a macro here to turn on the workaround on MI200 only. Like
`
#if defined(__gfx90a__)
 using ABDataTypeAdjusted = conditional_t<is_same_v<ABDataType, ck::half_t>, ck::bhalf_t, ABDataType>;
#esle
 using ABDataTypeAdjusted = ABDataType;
#endif 
`

Reply:
Done in [3360a1f](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/603/commits/3360a1f5c36af0a8d3e88afc5bf7776d762fee75)